models/stg_zendesk__ticket.sql

```diff
@@ -21,6 +21,13 @@ fields as (
                 staging_columns=get_ticket_columns()
             )
         }}
+
+        --The below script allows for pass through columns.
+        {% if var('zendesk__ticket_passthrough_columns',[]) != [] %}
```

Comment:
hey @fivetran-joemarkiewicz! I just tried this line the way it is and this way:
`{% if var('zendesk__ticket_passthrough_columns') != [] %}`, looks like this way works as well and may be less bulky!

Reply:
Good idea! I want to keep the current form however just to make sure we have a default value passed through.